src/pages/sa-clients/forms/ClientsAddressForm.js

```diff
@@ -0,0 +1,40 @@
+import React, { useContext, useEffect, useState } from 'react'
+import { RequestsContext } from 'src/providers/RequestsContext'
+import toast from 'react-hot-toast'
+import { ControlContext } from 'src/providers/ControlContext'
+import AddressForm from 'src/components/Shared/AddressForm'
+import { SaleRepository } from 'src/repositories/SaleRepository'
+import { useInvalidate } from 'src/hooks/resource'
+
+const ClientsAddressForm = ({ getAddressGridData, clientId, recordId, window, props }) => {
+  const [address, setAddress] = useState()
+  const { postRequest } = useContext(RequestsContext)
+  const { platformLabels } = useContext(ControlContext)
+
+  const onSubmit = async obj => {
+    if (obj) {
+      const data = {
+        clientId: clientId,
+        address: obj,
+        addressId: obj.recordId
+      }
+
+      await postRequest({
+        extension: SaleRepository.Address.set,
+        record: JSON.stringify(data)
+      }).then(res => {
+        if (!obj.recordId) {
+          toast.success(platformLabels.Added)
+        } else {
+          toast.success(platformLabels.Edited)
+        }
+        window.close()
+      })
+    }
+    getAddressGridData(clientId)
+  }
+
+  return <AddressForm {...{ ...props, address, setAddress, recordId: recordId, onSubmit }} />
```

Comment:
![image](https://github.com/user-attachments/assets/8c0d5ebf-1be5-499f-ad86-e5ccc64bbda2)
should't the name be mandatory here?